input/system/selfhosted/log_receiver.go

```diff
@@ -103,20 +103,17 @@ func DiscoverLogLocation(servers []*state.Server, globalCollectionOpts state.Col
 		}
 
 		if loggingCollector == "on" {
-			logDirectoryBytes, err := exec.Command("/usr/bin/pganalyze-collector-helper", "log_directory").Output()
+			logDirectory, err := getPostgresSetting("log_directory", server, globalCollectionOpts, prefixedLogger)
 			if err != nil {
-				prefixedLogger.PrintError("ERROR - Could not run helper process: %s", err)
+				prefixedLogger.PrintError("ERROR - Could not retrieve log_directory setting from Postgres: %s", err)
 				continue
 			}
-			logDirectory := string(logDirectoryBytes)
-			if logDirectory == "" {
-				prefixedLogger.PrintError("ERROR - Could not retrieve log_directory setting from Postgres")
-				continue
-			}
-			if !strings.HasPrefix(logDirectory, "/") {
-				logDirectory = status.DataDirectory + "/" + logDirectory
+
+			if strings.HasPrefix(logDirectory, "/") {
+				prefixedLogger.PrintInfo("Found log location, add this to your pganalyze-collector.conf in the [%s] section:\ndb_log_location = %s", server.Config.SectionName, logDirectory)
+			} else {
+				prefixedLogger.PrintInfo("WARNING: found log location in data directory - please check our setup guide for instructions.\n")
 			}
```

Comment:
@lfittl do you think hiding the result in this case is reasonable? Or should we still show it, but indicate you may need to make changes? My understanding is that this is only likely to work with pretty questionable permissions setups to begin with, but maybe it's worth keeping anyway?

Reply:
Maybe we can include it in the warning message, like this:

```
WARNING: Found relative log location "log" inside data directory - please check our setup guide for instructions\n
```

(I suspect this will be the default, i.e. `log`, 95% of the time)